pages/views/sitemap.njk

```diff
@@ -0,0 +1,17 @@
+---
+permalink: "/sitemap.xml"
+eleventyExcludeFromCollections: true
+---
+
```

Comment:
Cosmetic: please remove empty space here, it is better to have
`<?xml version="1.0" encoding="UTF-8"?>`
at the very beginning of the xml document (some riders can assume that as a warning)

Reply:
Done